src/argilla/server/contexts/datasets.py

```diff
@@ -803,10 +920,14 @@ async def delete_response(db: "AsyncSession", search_engine: SearchEngine, respo
     return response
 
 
-def validate_response_values(dataset: Dataset, values: Dict[str, ResponseValue], status: ResponseStatus):
+def validate_response_values(
```

Comment:
If it's only used internally:

```suggestion
def _validate_response_values(
```